yt_dlp/extractor/maariv.py

```diff
@@ -0,0 +1,72 @@
+from yt_dlp.extractor.common import InfoExtractor
+import re
+
+class MaarivIE(InfoExtractor):
```

Comment:
Thanks for your PR :)
Please add a 2nd blank line over here, so that the automated linter passes

NOTE: This is not a review, because I don't review PRs. I am just informing you about an issue that the automated linter found, so that that issue doesn't hold back whoever does review this PR